server/views/taskList/taskListTemplate.pug

```diff
@@ -33,7 +33,7 @@ block content
   if !user.role
     p Error
   else
-    if errors 
+    if errors && errors.length > 0
```

Comment:
you don't need to check for errors as it will always be present - as we represent no errors as an empty array